### ISSUE TITLE
Fix of "Failed to retrieve wine (lutris-GE-Proton8-10-x86_64) information"

### ISSUE THREAD

**Issue by jarrett-m** [TormStorm/jagex-launcher-linux#56]
After following the steps and trying to install via the script, I got the error "Failed to retrieve wine (lutris-GE-Proton8-10-x86_64) information"

This is because the latest version of lutris-GE-Proton does not match what is in the script.

To fix:

Edit the install script on line 21 from:
    version: lutris-GE-Proton8-10-x86_64
to
    version: lutris-GE-Proton8-12-x86_64 (or whatever the current proton version is that you installed)

**Comment by jarrett-m** [comment]
> I believe removing the "version:" line altogether should also grab the latest supported runner that Lutris puts out.
> 
> 
> 
> Somewhat related:
> 
> At present, as I understand it, we're having mixed results version to version with Wine runners and Lutris, across different platforms. Lutris is supposed to allow us to specify a given runner by specifying "version" (one that we can test for compatibility & stability across multiple platforms), but for some reason it fails to work sometimes with older (non-latest) runners.
> 
> 
> 
> As such, it's a fight between getting consistent results and staying up to date with Lutris' latest runner. The runner listed in the install script is usually the latest one we've tested and found consistent - though I think it's due for an update to include 8-12. I'll prod Torm on this.

Ah that makes sense, I don't mess with proton too much. This is far from a full QA test, but with the latest version of proton I am able to play osrs on RuneLite.

**Comment by nmlynch94** [comment]
Thanks for the PR. To add some context, we used to not have a version listed because it's supposed to default to the latest proton-GE version. This broke in the newest update so now it defaults to system wine unless the user has manually downloaded proton-ge. They have started iterating proton versions very quickly now so we updated the guide to instruct the person to take this manual step instead of manually keeping it up to date.
